src/test/java/uk/gov/pay/directdebit/payments/services/GoCardlessServiceTest.java

```diff
@@ -133,6 +137,25 @@ public void shouldValidateBankAccountDetails_ifExceptionIsThrownFromGC() {
         assertThat(response.getBankName(), is(nullValue()));
     }
 
+    @Test
+    public void shouldReturnNotEmptyServiceUserNameOptional_whenBacsSchemeExists() {
```

Comment:
`NotEmpty` reads a bit weirdly (`NonEmpty` would be more idiomatic).

At this level, we don’t really care that the reason we have a SUN name is because it’s Bacs, it’s simply because the service below had one. So maybe call this `shouldReturnSunNameForCreditorId`…